configs/storybook/src/createConfig.ts

```diff
@@ -0,0 +1,31 @@
+import type { StorybookConfig } from "@storybook/react-vite";
+
+type GetAbsolutePath = (input: string) => string;
+
+const createConfig = (getAbsolutePath: GetAbsolutePath): StorybookConfig => ({
+  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
```

Comment:
Is this ../src opinionated? Should users be able to pass in their own story directories perhaps?